underfs/s3/src/main/java/tachyon/underfs/s3/S3UnderFileSystem.java

```diff
@@ -347,6 +333,23 @@ private boolean deleteInternal(String key) {
   }
 
   /**
+   * Gets the child name based on the parent name. If the parent is not valid, the entire child
```

Comment:
Oh I see. then probably you shall update its javadoc accordingly as this is not very obvious.